src/main/java/core/basesyntax/BooleanExpression.java

```diff
@@ -14,6 +14,11 @@
      * без использования конструкций if-else</p>
      */
     public boolean booleanExpression(boolean a, boolean b, boolean c, boolean d) {
-        return false;
+        int a1 = a ? 1 : 0;
```

Comment:
read [checklist](https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_01.html) attentively

Use only boolean operators without if-else or **ternary operator**

read some information about [boolean operations](https://www.dummies.com/programming/java/logical-operators-in-java/)

Reply:
sorry for misunderstanding
done it correct way